### ISSUE TITLE
GitHub Issue and Pull Request templates

### ISSUE THREAD

**Issue by ntwb** [stylelint/stylelint#762]
GitHub announced today [Issue and Pull Request templates](https://github.com/blog/2111-issue-and-pull-request-templates) ( technical docs [here](https://help.github.com/articles/helping-people-contribute-to-your-project/)):
> It's hard to solve a problem when important details are missing. Now project maintainers can add templates for Issues and Pull Requests to projects, helping contributors add the right details at the start of a thread. This is the first of many improvements to Issues and Pull Requests that we're working on based on feedback from the community.

So here's a *rough* idea of what an issue template for stylelint might want to look like:

```md
### Issue description: Is this a [ ] bug, a [ ] rule enhancement, or a [ ] new rule? 

### Rule: Which rule is this issue related to? `insert-rule-name-here`

### CSS: Example CSS to help us reproduce your issue

.foo {
  color: pink;
}


### Configuration: What stylelint configuration are you using? 

{
  "rules": {
    "indentation": ["tab", {
      "except": ["value"]
    }]
  }
}

### Actual Behaviour: What are the warnings or errors you are getting?

### Expected Behaviour: What did you expect to see?

### Note: [Stackoverflow](http://stackoverflow.com/questions/tagged/stylelint) is our preferred QA forum
```

- [ ] Update and move `CONTRIBUTING.md` to `.github/CONTRIBUTING.md`
- [x] Create and add `.github/ISSUE_TEMPLATE`
- [ ] Create and add `.github/PULL_REQUEST_TEMPLATE `

**Comment by jeddy3** [comment]
Excellent! 

The questions will be beneficial for things like https://github.com/stylelint/stylelint/issues/760 and the stackoverflow bit for issues like https://github.com/stylelint/stylelint/issues/759.

I'll knock up a PR now.
